backend/app/project_handler_test.go

```diff
@@ -0,0 +1,263 @@
+package app
```

Comment:
table based tests may need some changes as you are not checking if the db contains the data you sent or updated or not, consider altering the structure or not using table based tests